tests/test_hypersphere.py

```diff
@@ -396,6 +398,24 @@ def test_mean_and_belongs(self):
         result = self.metric.mean([point_a, point_b, point_c])
         self.assertTrue(self.space.belongs(result))
 
+    def test_sample_von_mises_fisher(self):
+        """Check that the maximum likelihood estimate of the mean is
+        close to the real values for a large concentrated sample.
+        """
+        dim = 2
+        n_points = 1000000
+        sphere = Hypersphere(dim)
+
+        kappa = 1000000
+        points = sphere.random_von_mises_fisher(kappa, n_points)
+        sum_points = gs.sum(points, axis=0)
+        mean = gs.array([0, 0, 1])
+        mean_estimate = sum_points/gs.linalg.norm(sum_points)
```

Comment:
Here there should be a space round the operator, right?

Reply:
Yes that is completely right !